packages/dapp-svelte-wallet/api/src/types.js

```diff
@@ -98,7 +98,7 @@
 
 /**
  * @typedef {Object} RecordMetadata
- * @property {number} sequence
+ * @property {number} id
  * A monotonically increasing number to allow total ordering between records.
```

Comment:
Sorry, should have caught this before.

```suggestion
 * Identifies a particular Record in the context of the wallet backend.  This id is stable and unique for a wallet backend (even across different record types).
```

Reply:
Fixed